src/views/Recents/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default } from './RecentsView';
+export { fetchRecents } from './services/fetchRecents';
+export { fetchRecentsThunk, fetchRecentsThunkExtraReducers } from './services/fetchRecentsThunk';
```

Comment:
Separate the thunk from services, the thunk is related with redux, add it to store folder in Recents to separate it from services. 
Also,  the current barrel export structure could be improved. Right now, it's mixing view and service exports in a single file, create a barrel for each responsibility, better to not mix them :)